src/wing/CCPACSWingShell.cpp

```diff
@@ -99,4 +108,86 @@ TiglLoftSide CCPACSWingShell::GetLoftSide() const
     throw CTiglError("Cannot determine loft side, this shell is neither lower nor upper shell of parent");
 }
 
+bool CCPACSWingShell::SparSegmentsTest(gp_Ax1 nNormal, gp_Pnt nTestPoint, TopoDS_Shape nSparSegments) const
```

Comment:
Could this function be rewritten as follows?:
 - ProjectPointOnShape -> nearest point in shape
 - value = (NearestPoint - Point).Dot(nNormal) // scalar product between axis and projected point
 - return value > 0

Reply:
If it still does the same....why not.